core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -14,9 +14,14 @@
     GLOBALS.canEdit = JSON.parse('{{can_edit|js_string}}');
     GLOBALS.canUnpublish = JSON.parse('{{can_unpublish|js_string}}');
     GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
+    GLOBALS.collectionVersion = JSON.parse('{{collection_version|js_string}}');
     GLOBALS.collectionTitle = JSON.parse('{{title|js_string}}');
     GLOBALS.isPrivate = JSON.parse('{{is_private|js_string}}');
     GLOBALS.TAG_REGEX = JSON.parse('{{TAG_REGEX|js_string}}');
+
+    GLOBALS.owners = JSON.parse('{{owners|js_string}}');
```

Comment:
Do you think we could get this info via an AJAX call, and not in GLOBALS? We're actually trying to remove the need for GLOBALS, long-term.